src/DotVVM.Framework/Resources/Scripts/DotVVM.ts

```diff
@@ -446,9 +446,9 @@ class DotVVM {
                 errorCallback({ xhr });
                 dotvvm.events.staticCommandMethodFailed.trigger({ ...data, xhr })
             },
-            xhr => {
-                xhr.setRequestHeader("X-PostbackType", "StaticCommand");
-            });
+                xhr => {
+                    xhr.setRequestHeader("X-PostbackType", "StaticCommand");
+                });
         })()
```

Comment:
Could you please not reformat code? The same file was changed in #787 and we'll later have to go through all the changes in JS and merge them into `js-modules`. This just makes merging very annoying.